preprocess/global.inc

```diff
@@ -13,4 +13,5 @@ function bixaluswds_preprocess(&$variables) {
   $language = Drupal::languageManager()->getCurrentLanguage();
   $variables['language'] = $language->getId();
   $variables['uswds_img_path'] = '/themes/custom/bixaluswds/dist/img';
+  $variables['class_prefix'] = 'bx-';
```

Comment:
**Question (non-blocking):** Curious why this is `bx` and not `bix`.

Reply:
I am hoping with will be configurable in the admin. For instance we are using `vote-` prefix on the vote.gov website. bx is just a default setting. I believe this was prefix Danny used for the original theme.